plenoptic/metric/perceptual_distance.py

```diff
@@ -387,18 +390,19 @@ def nlpd(IM_1, IM_2):
 
     Parameters
     ----------
-    IM_1: torch.Tensor
-        image, (1 x 1 x H x W)
-    IM_2: torch.Tensor
-        image, (1 x 1 x H x W)
+    IM_1: torch.Tensor of shape (1, 1, height, width)
+        The first image.
+    IM_2: torch.Tensor of shape (1, 1, height, width)
+        The second image.
 
     Returns
     -------
     distance: float
+        The normalized Laplacian Pyramid distance.
 
     Note
     ----
-    only accepts single channel images
+    The input images have to be single-channel.
```

Comment:
We may be able to support batch and multichannel by cleaning our current LaplacianPyramid implementation, which would simplify the regression on perceptual data - do you think it is worth it?

Reply:
I think it is, will implement batch and multichannel.